managed-databases/mongodb/reference-content/autohealing.mdx

```diff
@@ -0,0 +1,25 @@
+---
+meta:
+  title: Understanding the autohealing feature
+  description: Learn more about the autohealing feature for Managed MongoDB™ Databases
+content:
+  h1: Understanding the autohealing feature
+  paragraph: Learn more about the autohealing feature for Managed MongoDB™ Database
+tags: databases ha high-availability autohealing database-nodes
+categories:
+  - managed-databases
+---
+
+At Scaleway, High Availability Database Instances are continuously monitored and benefit from the autohealing feature.
```

Comment:
caps look a bit weird, maybe less weird if you write "High Availability (HA) Database Instances"?

Reply:
```suggestion
At Scaleway, High Availability (HA) Database Instances are continuously monitored and benefit from the autohealing feature.
```